src/webgl/context.js

```diff
@@ -218,21 +219,17 @@ export function withParameters(gl, {scissorTest, framebuffer, nocatch = true}, f
     if (framebuffer) {
       // TODO - was there any previously set frame buffer?
       // TODO - delegate "unbind" to Framebuffer object?
-      gl.bindFramebuffer(gl.FRAMEBUFFER, null);
+      framebuffer.unbind();
     }
   }
 
   let value;
-  if (nocatch) {
```

Comment:
Can you comment it out instead of deleting - I added it recently since all these catches can interfere with debugging so I want to disable it in places where I am reasonably confident that there will not be exceptions.